### PR TITLE
docs(readme): Update readme with new architecture

### PR DESCRIPTION
**Pull request by jchauncey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,50 +20,21 @@ The new v2 logger implementation has seen a simplification from the last rewrite
...
-
-The replication controller should immediately restart a new pod. Now you can use the `deis logs` command for your applications.
+* `DEIS_NSQD_SERVICE_HOST` - default:""
+* `DEIS_NSQD_SERVICE_PORT_TRANSPORT` - default:4150
+* `NSQ_TOPIC` - default:logs
+* `NSQ_CHANNEL` - default:consume
+* `NSQ_HANDLER_COUNT` - default:30
+* `AGGREGATOR_STOP_TIMEOUT_SEC` - default:1
+* `DEIS_LOGGER_REDIS_SERVICE_HOST` - default:""
+* `DEIS_LOGGER_REDIS_SERVICE_PORT` - default6379
```
Comment (mboersma) [inline README.md]: Needs a colon `:` here to separate "default" from "6379."

File: README.md [hunk 2]
```diff
@@ -87,13 +58,44 @@ DEV_REGISTRY=myhost:5000 make push
...
 ### Kubernetes interactions
-* `DEV_REGISTRY=quay.io IMAGE_PREFIX=myaccount make kube-create`: Does a sed replacement of the image name and creates a tmp manifest file that we will use to deploy logger component to kubernetes. This will also start 2 `fluentd` daemonsets.
-* `make kube-delete`: This will remove all the logger components from the kubernetes cluster.
-* `DEV_REGISTRY=quay.io IMAGE_PREFIX=myaccount make kube-replace`: This will rebuild the binary and image, push it to the remote registry, and then replace the running components with the new version.
+* `make install` - Install the recently built docker image into the kubernetes cluster
+* `make upgrade` - Upgrade a currently installed image
+* `make uninstall` - Uninstall logger from a kubernetes cluster
+
+### Architecture Diagram
+```
```
Comment (mboersma) [inline README.md]: This diagram needs some whitespace corrections so boxes align cleanly.
screen shot 2016-08-26 at 11 33 50 am

Comment (jchauncey) [inline README.md]: weird i dont see any formatting issues - 
screen shot 2016-08-26 at 1 50 26 pm

Comment (mboersma) [inline README.md]: @jchauncey if I just click ["View"](https://github.com/jchauncey/logger/blob/6596685324c00991a16e17713e9c8bc5f240aefa/README.md) on this PR I see the formatting issue in the Markdown. 😕

Comment (jchauncey) [inline README.md]: yeah im not sure whats up. Do you see any issues here https://github.com/deis/monitor#architecture-diagram

Comment (mboersma) [inline README.md]: Yes, I see the same formatting issues there.

Comment (helgi) [inline README.md]: Weird, I see it normal.

Comment (jchauncey) [inline README.md]: alright im not sure how to fix this then =\

Comment (mboersma) [inline README.md]: It must have to do with browser + font choice then. Probably it's too much to ask of Markdown that a preformatted section align perfectly so a diagram like this always works.

All I can think of as a workaround is replacing it with an actual image. 🎨

**Pull request merged**